### PR TITLE
Fetch reviews and store in JSON file.

### PR DESCRIPTION
**Pull request by TomerMiz10**
## Task link from JIRA:
link: < (example: https://mssplinter10-1711638164183.atlassian.net/jira/software/projects/FDP/boards/1?selectedIssue=FDP-15)>

## Description:
Add TripAdvisor API service. The service makes a GET request to the TripAdvisor API to fetch restaurant reviews and saves the response data to a JSON file. Also, basic testing.

### CODE REVIEW

File: backend/src/api/tripadvisorAPI.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      headers: {
+        'X-RapidAPI-Key': 'f8662f8ed1msh343418130ebdb98p11bf57jsna9bf16098028',
+        'X-RapidAPI-Host': 'tripadvisor16.p.rapidapi.com'
+      }
+};
+
+const fetchReviews = async () => {
+    try{
+        const response = await axios.request(options);
+        fs.writeFileSync(reviewsFilePath, JSON.stringify(response.data.data.data, null, 2));
```
Comment (nitzanto) [inline backend/src/api/tripadvisorAPI.service.ts:21]: Try doing doing with JSON.stringify only

File: backend/src/api/tripadvisorAPI.service.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        console.log('File written successfully');
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+export default { fetchReviews };
```
Comment (nitzanto) [inline backend/src/api/tripadvisorAPI.service.ts:28]: Why isnt this in the services folder? You can move that api folder there.

Also what about the model for restaurants and posts? The data fetched from the API should be manipulated to fit the model and for easy use.

File: backend/src/api/reviews.json
Comment (nitzanto) [inline backend/src/api/reviews.json:1]: This should be manipulated to fit a proper structure that's easy to read & use across the app.

**Review by nitzanto — CHANGES_REQUESTED** [review]
Please read the comments but this seems to be in the right direction.

File: backend/src/api/tripadvisorAPI.service.ts (hunk 2 above)
Comment (TomerMiz10) [reply backend/src/api/tripadvisorAPI.service.ts:28]: the next branch with the next PR will have this functionality.
i need to figure out how to manipulate that data into the DB, with the corresponding models.
also, develop that with TDD.

File: backend/src/api/reviews.json
Comment (TomerMiz10) [reply backend/src/api/reviews.json:1]: next Jira task and PR

**Pull request closed without merging**